daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -193,7 +200,7 @@ class PrimitiveTunable(name: String, schemaType: String, schemaDefault: String)
   }
 
   private val scalaDefault = schemaType match {
-    case "xs:string" => "\"" + schemaDefault + "\""
+    case "xs:string" if (name != "tempFilePath") => "\"" + schemaDefault + "\""
     case _ => schemaDefault
   }
```

Comment:
What happens if someone does someting like this:
```scala
dp.setTunable("tempFilePath", "/custom/path")
...
dp.setTuanble("tempFilePath", "")
```
So they effectively want to set it back to the default? Since the ``scalaDefault`` change only affects the initial value of the tunable, the second setTunable call won't revert back to java.io.tmpdir. Which is potentially reasonable, but what will the behavior be? If the Path is the empty, will that fail, will it be the current directory, or something else? Not sure I feel strongly about what happens, but just want to make sure whatever hapens is reasonable.